apps/src/code-studio/components/libraries/libraryParser.js

```diff
@@ -36,7 +36,8 @@ function createDropletConfig(functions, libraryName) {
     let individualConfig = {
       func: currentFunction.functionName,
       category: 'Functions',
-      comment: currentFunction.comment
+      comment: currentFunction.comment,
+      type: 'either'
```

Comment:
For documentation purposes: Mind leaving a comment on this PR showing where you found this?

Reply:
Sure so the relevant droplet code is here: https://github.com/droplet-editor/droplet/blob/code-dot-org/src/languages/javascript.coffee#L211
```
if not known or (known.fn.value and known.fn.command)
  return [node.type, 'any-drop']
```
Which basically says if `value` and `command` are both set to true, allow the block to drop into any kind of socket (either value or top-level expression).

On our end, the relevant code is: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/dropletUtils.js#L776
```
case 'either':
  newFunc.value = true;
  newFunc.command = true;
```
which is just a small convenience so we can write `type: 'either'` in the dropletConfig instead of specifying both `value` and `command`